src/components/Banners/SiteBanner.tsx

```diff
@@ -27,16 +25,21 @@ export const SiteBanner = () => {
 
   return (
     showBanner && (
-      <DismissibleBanner bannerTheme={theme} data-cy="sitewide-banner">
-        <span>{jiraLinkify(text, jiraHost)}</span>
-        <IconButton
-          aria-label="Close Site Banner"
-          onClick={hideBanner}
-          data-cy="dismiss-sitewide-banner-button"
-        >
-          <Icon glyph="X" />{" "}
-        </IconButton>
-      </DismissibleBanner>
+      <Banner
+        data-cy="sitewide-banner"
+        dismissible
+        onClick={hideBanner}
+        variant={mapThemeToVariant?.[theme] ?? Variant.Info}
```

Comment:
I wanna say it wouldn't crash if theme is `undefined` or not a valid entry, but I would double check
```suggestion
        variant={mapThemeToVariant[theme] ?? Variant.Info}
```